### DISCUSSION TITLE
Akash Community Pool & Inflation Proposal - July 2023

### DISCUSSION THREAD

**Discussion by lechenghiskhan** [akash-network/community#272] [Economics]
## Background and Context

The first set of major AKT 2.0 upgrades (Stable Payments & Take Rates) is going live with Akash Network’s Mainnet 6 upgrade in August. Until the Incentive Distribution Pool and its sub-accounts are completed ([discussion](https://github.com/orgs/akash-network/discussions/168)), the take rate proceeds will be funnneled into the existing Community Pool and will cover everything from network bounties and grants to community rewards and incentives.

With the GPU marketplace going live within the aforementioned Mainnet 6 upgrade in August, provider and deployment [incentives](https://gist.github.com/gosuri/f252b366c0cbe8925cf0666a3ad3a411#provider-subsidies) will become increasingly important for bootstrapping the GPU network during its infancy. 

During the early stages of the network’s GPU marketplace development, demand will likely not be attractive for providers to list high-density GPUs on Akash without some degree of utility guarantees.

## Timeline

Estimated GH discussion post date: July 13, 2023 
Estimated on-chain submission date: July 19, 2023
Estimated go-live date: July 26, 2023 

## Proposal

To solve the early-stage growing pains of the GPU marketplace, lay the foundation for growth, and until the IDP and other accounts are set up, we propose using the community pool to offset the lack of demand through targeted incentives. 

As demands on the Community Pool grow, so too should its funding. To that end, we propose two changes to be made concurrently. The estimated calculations can be viewed [here](https://docs.google.com/spreadsheets/d/1wm2ESNOndQdjASUuDEk4n_iHXnnYX8iqPjwomVzbnGg/edit#gid=0). 

1. Increasing the community pool tax from 10% to 25% to grow the community pool and ensure it is properly funded for provider and deployer incentives.  

2. Increase inflation min from 5% to 8% in order to increase effective inflation and offset the economic impact on validators and stakers created by the increased tax. This will maintain incentives for securing the network. Please note that the actual impact to effective inflation and staking APY are purely estimates and will depend on network-wide staking behavior. 

The use of community pool proceeds are and always will be submitted via governance proposals and determined by detailed discussions within sig-economics, steering committee, and associated working group meetings. 

Please join us at the next sig-economics meeting at 10 AM Pacific on August 2nd, 2023 where we’ll discuss the intended use of community pool proceeds, answer questions, and address any thoughts from the community.

**Comment by hewi333** [comment]
I know there has been some discussion in Telegram regarding how this affects max supply of AKT - Correct me if I am wrong, but all that increasing inflation does is shorten the time it takes to reach max supply? For example (I am going to make numbers up), if current inflation would result in max supply being reached in 10 years (2033), this may cause max supply to happen in 9 years (2032).

**Comment by SophoclesSummer** [comment]
This proposal was approved on 7/27/2023 by a 92% majority. 
https://www.mintscan.io/akash/proposals/211
